tutorials/tools/SDE_HowTo_v2.ipynb

```diff
@@ -45,19 +55,26 @@
       "execution_count": null,
       "id": "c3919489",
       "metadata": {
-        "id": "c3919489"
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 1000
+        },
+        "id": "c3919489",
+        "outputId": "e1b8d57f-854c-4d38-b09f-dda47e7c9214"
       },
       "outputs": [],
       "source": [
-        "BRANCH = 'main'\n",
+        "BRANCH = 'r2.6.0'\n",
         "\n",
         "!git clone -b $BRANCH https://github.com/NVIDIA/NeMo\n",
         "\n",
-        "!apt-get update && apt-get install -y libsndfile1 ffmpeg\n",
-        "!cd NeMo;./reinstall.sh\n",
-        "!apt-get install sox\n",
+        "!apt-get update && apt-get install -y libsndfile1 ffmpeg sox\n",
+        "!pip3 install -e ./NeMo/\n",
         "\n",
-        "!pip3 install -r ./NeMo/tools/speech_data_explorer/requirements.txt"
+        "!pip3 install -r ./NeMo/tools/speech_data_explorer/requirements.txt\n",
+        "!pip3 install -r ./NeMo/requirements/requirements_asr.txt\n",
+        "!pip3 install -r ./NeMo/requirements/requirements.txt\n",
+        "!pip3 install -r ./NeMo/requirements/requirements_lightning.txt"
```

Comment:
do we need this? Aren;t they already satisfied with 
```
pip3 install -e ./NeMo/\
```
above

Reply:
`pip install -e` only installs ./requirements/main.txt